src/commands/create/groups/quickstart.ts

```diff
@@ -99,17 +127,33 @@ export default async (folderLocation: string, folderRelativePath: string, templa
     ...env,
     WALLET_PRIVATE_KEY: privateKey,
   };
-  const packageManager = await askForPackageManager();
+
+  const packageManager: PackageManagerType = template.framework === "Foundry" ? "forge" : await askForPackageManager();
   await setupTemplate(template, folderLocation, env, packageManager);
 
   successfulMessage.start(folderRelativePath);
-  Logger.info(`${chalk.magentaBright("Directory Overview:")}
-  - Contracts: /contracts
-  - Deployment Scripts: /deploy
-   
-${chalk.magentaBright("Commands:")}
+  // Define directory paths based on the framework
+  const isFoundry = template.framework === "Foundry";
```

Comment:
Let's maybe split foundry and others into different functions?
So we have easier to read code:
```ts
if (template.framework === "Hardhat") {
   logHardhatInfo(...);
} else if (template.framework === "Foundry") {
   logFoundryInfo(...);
}

Reply:
then you don't need to insert `isFoundry` everywhere and it would be a bit more scalable